.yamllint.yml

```diff
@@ -26,3 +26,5 @@ rules:
   braces:
     min-spaces-inside: 1
     max-spaces-inside: 1
+  trailing-spaces: disable
+  new-line-at-end-of-file: disable
```

Comment:
I once worked on a team that would break builds if source files didn't end with a newline.  The reasoning was that it'd be unnecessary diff change if we ever added anything to the file.  Rather than have future diff noise, we just mandated that in the style guide and enforced it with a simple linter check.  I feel like I have a strong opinion based on that experience, but I guess I don't.  I feel like using terminal newlines is good practice, but I don't need linters to break builds like they once did to me.